TestSamples/SwiftUITestSample/SwiftUITestSampleUITests/XCUIApplication+uninstall.swift

```diff
@@ -0,0 +1,48 @@
+import XCTest
+
+// This snipped was obtained from https://www.jessesquires.com/blog/2021/10/25/delete-app-during-ui-tests/
+extension XCUIApplication {
+    func uninstall(name: String? = nil) {
```

Comment:
Nice! We could use this for all flaky tests where we need a clean state instead of using the `--io.sentry.wipe-data` to simulate a cleanup

Reply:
Whoa, this is an awesome find 👍🏻 👍🏻